client/packages/system/src/Item/Components/StockItemSearchInput/StockItemSearchInput.tsx

```diff
@@ -100,6 +124,28 @@ export const StockItemSearchInput: FC<StockItemSearchInputProps> = ({
           'name'
         ).sort((a, b) => a.label.localeCompare(b.label))
       }
+      inputValue={search}
+      inputProps={{
+        onChange: e => {
+          const { value } = e.target;
+          setSearch(value);
+          if (!!currentItem) {
```

Comment:
Current item doesn't seem to exist for me when I select an item but it does exist after I try to change the item...